bot.js

```diff
@@ -10,13 +10,23 @@ client.on('ready', function() {
 	console.log(`Logged in as ${client.user.tag}!`);
 });
 
+// reaction roles code
+const reaction_bot = require("./reaction_roles.js")
+// check every event for reaction add/remove on the reaction role message.
+client.on("raw", event => {
```

Comment:
Unless necessary (i.e. to avoid rebinding of `this`) arrow functions are discouraged for anything more than a quick one-line filter or transform. Avoid arrow functions for anything more than a one-liner.

i.e.
```
var f = function(a, b) { return a + b; }
```

Can be re-written as:
```
var f = (a, b) => a+b;
```

But not:
```
var f = function(a, b) {
	if (a > b)
		return a - b;
	else
		return b - a;
}
```
```suggestion
client.on("raw", function(event) {
```